Sources/AnimalKingdomAPI/AnimalKingdomAPI/Schema/CustomScalars/CustomDate.swift

```diff
@@ -0,0 +1,6 @@
+// @generated
+// This file was automatically generated and should not be edited.
```

Comment:
We need to give this a different header comment, but that can be done in a follow up PR.

Reply:
Doh!